.github/workflows/pr_assignee.yml

```diff
@@ -1,19 +1,19 @@
 name: Assign PR to creator
 
-# Due to GitHub token limitation, only able to assign org members not authors from forks.
-# https://github.com/thomaseizinger/assign-pr-creator-action/issues/3
-
 on:
   pull_request:
```

Comment:
According to the documentation this needs to be pull_request_target

Reply:
```suggestion
  pull_request_target:
```

Reply:
Then maybe milestone should go to `pull_request_target` too?

Reply:
It's specified in the auto assign action readme + issue, no mention of it for the milestone action.

Reply:
It is mentioned here

https://github.com/andrefcdias/add-to-milestone/blob/399d78cf88a6d47d4b600e2e0b1f4f0b2bdfbe7c/src/runner.ts#L94

Reply:
Yeah ok, could you make update it with a new PR please. Also please remove the GitHub token line for auto assign while you are at it.